paimon-flink/paimon-flink-cdc/src/test/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSyncTableActionITCase.java

```diff
@@ -1055,6 +1055,7 @@ public void testOptionsChange() throws Exception {
         Map<String, String> tableConfig = new HashMap<>();
         tableConfig.put("bucket", "1");
         tableConfig.put("sink.parallelism", "1");
+        tableConfig.put("sequence.field", "_timestamp");
```

Comment:
I think we can add a new test method. In the new method, we can create a table first, then we try to build cdc action with table config. We can randomly put immutable config key to the table config, and the immutable config won't cause error.

Reply:
done.